articles/tech-resources/enabling-https-engagement-tracking-on-sparkpost.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Enabling HTTPS Engagement Tracking on SparkPost"
+description: "SparkPost supports HTTPS engagement tracking for customers via self-service for all SparkPost customers. To enable SSL engagement tracking for a domain, additional configuration for SSL keys is required."
+---
+
+## Overview
+
+SparkPost supports HTTPS engagement tracking for customers via self-service for all SparkPost customers. To enable SSL engagement tracking for a domain, additional configuration for SSL keys is required.
```

Comment:
I might also explain the data flow up front:

`After completing the steps below, your email recipients will see HTTPS links in the email you send. When they visit a tracked link, your CDN will handle the SSL connection, then pass the HTTP request on to SparkPost. SparkPost will record the engagement event and redirect the recipient to the original URL.`